src/Management/test/Endpoint.Test/Trace/TraceEndpointTest.cs

```diff
@@ -24,7 +24,7 @@ public TraceEndpointTest(ITestOutputHelper output)
     public void Constructor_ThrowsIfNullRepo()
     {
         IOptionsMonitor<TraceEndpointOptions> opts = GetOptionsMonitorFromSettings<TraceEndpointOptions>();
-        Assert.Throws<ArgumentNullException>(() => new TraceEndpoint(opts, null));
+        Assert.Throws<ArgumentNullException>(() => new TraceEndpoint(opts, null, null));
```

Comment:
I have the same concern as raised earlier. I think we should either complete this test or remove it.